carbon-design-system/carbon-addons-iot-react
# feat(dashboard-editor): add decimal precision to ImageCards

Closes #2045 

**Summary**

- Adds precision to data items on ImageCards in the DashboardEditor

**Change List (commits, features, bugs, etc)**

- display precision dropdown on ImageCards
- add precision to the reducer to save its state
- update styles to fix increment/decrement on thresholds
- update test cases to account for the extra buttons in the modal

**Acceptance Test (how to verify the PR)**

- Add an image card to the DashboardEditor
- Edit the image
- add Pressure as a data item
- edit Pressure
- Ensure Aggregate/Precision fields are available and persist after change/save
